app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,34 @@
+<nav class="navbar navbar-default navbar-fixed-top">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+      <span class="sr-only">Toggle navigation</span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand nav-logo" href="/">FancyFeed</a>
+    </div>
+    <% if current_user %>
+    <div class="navbar-left">
+      <%= form_tag(search_path, :method => "get", class: "navbar-form", id: "search-form") do %>
+        <div class="input-append">
+          <%= text_field_tag :query, params[:query], class: "span2", placeholder: "Search" %>
+          <button class="btn" type="submit"><i class="glyphicon glyphicon-search"></i></button>
+        </div>
+      <% end %>
+    </div>
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+      <ul class="nav navbar-nav navbar-right">
+        <li>
+          <%= link_to user_path(current_user), class: "btn menu-button" do %>
+            <span class="glyphicon glyphicon-cog" aria-hidden="true"></span>
+            <span class="menu-word">Settings</span>
+          <% end %>
+        </li>
+        <li><%= link_to "Log out", logout_path, method: :delete, class: "btn menu-button" %></li>
+        </ul>
+      <% end %>
```

Comment:
Be sure to line up your `<% if %>` and `<% end %>` tags so that it is clear what is included in the conditional.